bin/ytdl.js

```diff
@@ -141,8 +147,25 @@ function printVideoInfo(info) {
   console.log(label('length: ') + util.toHumanTime(info.length_seconds));
 }
 
+/**
+ * Prints basic video information as JSON.
+ *
+ * @param {Object} info
+ */
+function printVideoInfoJson(info) {
+  console.log(JSON.stringify(info));
```

Comment:
I think we can do without this function and embed the contents in the line below.

`printVideoInfo` is a function because it's called from two places.